### DISCUSSION TITLE
MCBBS 信息打开后报错？

### DISCUSSION THREAD

**Discussion by Ad-closeNN** [Meloong-Git/PCL#3215] [提问] [answered]
![PixPin_2024-01-28_17-33-34](https://github.com/Hex-Dragon/PCL2/assets/103355026/62252371-8c99-4235-ab79-1257aeb0756e)
如题，我像往常一样打开PCL，看见关于镜像源问题的PCL弹窗，我点击查看详情就提示如上图所示的提示
**没用另一台电脑试，打开另一个PCL也不弹这个提示**

**Comment by LTCatt** [comment, marked as answer]
已修复
